### ISSUE TITLE
Castle (inside) Map

### ISSUE THREAD

**Issue by saporter** [FortWorthGameDev/HeartHome#33]
Designer--> A scene for the inside of the castle.  To keep it simple, we need two rooms.  Only the first room is accessible when first entering; door to the second room is locked until mini game complete.  There should be a rear exit from the castle in the second room.
-->The first room is small with a painting on the wall that a player can interact with (to be taken to a mini game -- another issue for this).
-->Once that mini game is complete, door to second room unlocks.  The second room is a throne room.
(see https://docs.google.com/document/d/1pR9PLLrS4NxCNCBC2EqR4rsC7BtZ7ym85IjhC7at_sM/edit?ts=5aa946ee#bookmark=id.fhkuxij0p15k)

Design doc:
https://docs.google.com/document/d/1pR9PLLrS4NxCNCBC2EqR4rsC7BtZ7ym85IjhC7at_sM/edit?ts=5aa946ee#bookmark=id.23tz53ot1bp0
